### DISCUSSION TITLE
使用formily/antv 时全局导入ant-design-vue ，报错

### DISCUSSION THREAD

**Discussion by Picktarget** [alibaba/formily#2807] [Q&A 2.x] [answered]
复现链接 https://codesandbox.io/s/frosty-fast-66b1y
期待：全局导入ant-design-vue页面正常显示
结果：全局导入ant-design-vue页面没有内容，注释掉导入的内容后出现渲染的表单示例
![0FC88E66-6F89-4745-91DE-F5104334F9AA](https://user-images.githubusercontent.com/12743709/152900692-765c8159-e2d4-419d-93d9-ac4e80e5ae50.png)

**Comment by zhouxinyong** [comment, marked as answer]
pr https://github.com/alibaba/formily/pull/2810 已修复
